Mathlib/RingTheory/DedekindDomain/PID.lean

```diff
@@ -42,7 +42,6 @@ theorem Ideal.eq_span_singleton_of_mem_of_not_mem_sq_of_not_mem_prime_ne {P : Id
     exact hxP2 (zero_mem _)
   by_cases hP0 : P = ⊥
   · subst hP0
-    -- Porting note: was `simpa using hxP2` but that hypothesis didn't even seem relevant in Lean 3
     rwa [eq_comm, span_singleton_eq_bot, ← mem_bot]
```

Comment:
Does `simpa` work here?

Reply:
Only with some tweaking to the `eq_comm` and the `mem_bot` so it rewrites the right things:
```suggestion
    simpa [eq_comm (a := ⊥), span_singleton_eq_bot, mem_bot] using x_mem
```
(Feel free to apply this change if you think it really is better.)

Reply:
I see. In that case, the current proof is fine!